src/databricks/sql/auth/oauth.py

```diff
@@ -14,17 +14,19 @@
 from requests.exceptions import RequestException
 
 from databricks.sql.auth.oauth_http_handler import OAuthHttpSingleRequestHandler
+from databricks.sql.auth.endpoint import OAuthEndpoints
 
 logger = logging.getLogger(__name__)
 
 
 class OAuthManager:
-    OIDC_REDIRECTOR_PATH = "oidc"
-
-    def __init__(self, port_range: List[int], client_id: str):
+    def __init__(
+        self, port_range: List[int], client_id: str, idp_endpoint: OAuthEndpoints
+    ):
         self.port_range = port_range
         self.client_id = client_id
         self.redirect_port = None
+        self.idp_endpoint = idp_endpoint
```

Comment:
By changing this function signature does this break the implementation for anyone using AWS after they upgrade their version of this connector?

Reply:
`OAuthManager` is only internally used by the PySQL. The consumer of the SDK uses the API `connect` in `databricks/sql/client.py`, and it does not have `endpoint` parameter